src/scicloj/02_numpy.clj

```diff
@@ -28,12 +30,21 @@ Python Data Science - ch.2. NumPy translated to Clojure
 In Clojure we use [dtype-next](https://github.com/cnuernber/dtype-next) (also known as
 `tech.v3.datatype`) and the convenient wrapper with a consistent API, [tablecloth](https://scicloj.github.io/tablecloth/),
 for working efficiently with dataset data."]
-
 (require
-  '[tech.v3.dataset :as ds]
-  '[tech.v3.datatype :as dtype]
-  '[tech.v3.tensor :as dtt]
-  '[tablecloth.api :as api])
+ '[tech.v3.dataset :as ds]
+ '[tech.v3.datatype :as dtype]
+ '[tech.v3.tensor :as dtt]
+ '[tablecloth.api :as api])
+
+^k/hidden
+(declare index)
+^k/hidden
+(defn ravel
+  "See `numpy.ravel`"
+  [t]
+  (let [shape (dtype/shape (dtt/->tensor t))
+        ix    (index shape)]
+    (dtype/make-reader :object (apply * shape) (apply dtt/mget t (nth ix idx)))))
```

Comment:
This could simply be: `(defn ravel [t] (dtype/->buffer t))`.

Reply:
Or, if you want to stay in tensor-land use `(dtt/reshape t (dtype/ecount t))`.  Both will result in an unraveled, 1D vector.

Reply:
Changed to `->reader`.  Maybe `->buffer` would be better?  Never used a buffer before.